docker-compose.yml

```diff
@@ -17,17 +18,59 @@ services:
       - "10001:10001"
       - "10002:10002"
 
+  next-client:
+    image: carrot-next-client
+    build:
+      context: app/next-client-app
+      dockerfile: Dockerfile
+    ports:
+      - 3000:3000
+    environment:
+      - BACKEND_URL=http://web:8000
+      - BACKEND_ORIGIN="web:8000"
+    volumes:
+      - ./app/next-client-app:/app/next-client-app
+
+  omop-lite:
+    image: ghcr.io/andyrae/omop-lite
+    volumes:
+      - ./vocabs:/vocabs
+    depends_on:
+      - db
+    environment:
+      - DB_PASSWORD=postgres
+      - DB_NAME=omop
+
   web:
     image: carrot
     build:
       context: app
       dockerfile: Dockerfile
     ports:
       - 8000:8000
-    env_file:
-      - .env
+    environment:
+      - NEXTJS_URL=http://next-client:3000
+      - ALLOWED_HOSTS=['localhost', '127.0.0.1','web']
+      - COCONNECT_DB_ENGINE=django.db.backends.postgresql
+      - COCONNECT_DB_HOST=db
+      - COCONNECT_DB_PORT=5432
+      - COCONNECT_DB_NAME=omop
+      - COCONNECT_DB_USER=postgres
+      - COCONNECT_DB_PASSWORD=postgres
+      - COCONNECT_DB_AUTH_TOKEN=
+      - UPLOAD_ONLY=false
```

Comment:
a couple of these are no longer needed:

```
COCONNECT_DB_AUTH_TOKEN
UPLOAD_ONLY
NLP_QUEUE_NAME
SCAN_REPORT_QUEUE_NAME
CREATE_CONCEPTS_QUEUE_NAME
CCOM_APP_URL
```

I would check `api/config/settings.py` for some that might be missing - we have sensible defaults that mostly assume the app is running on localhost.